src/ultrastardx.dpr

```diff
@@ -71,6 +71,7 @@ uses
 
   zlib                   in 'lib\zlib\zlib.pas',
   freetype               in 'lib\freetype\freetype.pas',
+  iconvenv               in 'lib\iconvenc\iconvenc.pas',
```

Comment:
typo: iconvenv != iconvenc

Reply:
Well spotted. Fixed

Reply:
I don't see any fix.

Why do we use our own copy of that unit on platforms where Free Pascal does provide the unit? It would simplify the code if we removed everything that isn't used on Windows.

Reply:
Ah, good pooint. Did not consider that. Trimmed the file down now to hopefully include only what we actually need for Windows, and use the provided on other platforms.